### ISSUE TITLE
Accessibility: View item page missing table cell

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#586]
The Related Links table is missing a cell, and it's being flagged by SiteImprove. 

Also, the table headers should be H2's (this was my bad. sorry!) [Details, Files, Title and contact, Contributors, Dates, Citation, Abstract and keywords, Type of deposit, Related content, Access settings, License and additional terms of use]

**Comment by justinlittman** [comment]
@astridu Can you still reproduce "related Links table is missing a cell"? It may have already been fixed.
